### PR TITLE
Events + activityTypes

### PR DESCRIPTION
**Pull request by psrpinto**
All that is here is merged, except for:

- `location` addition is waiting for approval: https://github.com/teamleadercrm/core/pull/3373

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -231,6 +231,132 @@ Get the current authenticated user.
...
+### events.list [GET /events.list]
+
+Get a list of calendar events.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + ids: `255a988d-d4b6-47eb-a5b8-746c24b5171a`,`e41cf7cd-05cd-4f28-b138-d6d585c30c64` (array[string], optional)
+            + user_id: `5309c8c9-e313-47b0-90ba-6850c3dc3e33` (string, required)
```
Comment (stivni) [inline apiary.apib]: @jenssegers @regularjack if this is required, is it in fact still a filter then? would it make sense to pass it one level up?

Comment (jenssegers) [reply apiary.apib]: I see it as a filter that *could* become optional down the road.

**Review by jenssegers — APPROVED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -231,6 +231,138 @@ Get the current authenticated user.
...
+                + creator (object)
+                    + type: `user` (string)
+                    + id: `e51d1a0e-452e-4d5d-a2a1-2c8f75dde560` (string)
+                + todo (object)
+                    + type: `todo` (string)
+                    + id: `eb399573-e1ae-4377-bbaf-34358eb19831` (string)
+                + activity_type (object)
+                    + type: `activityType` (string)
+                    + id: `28158af3-2ef3-4178-ac8b-2542b1481af0` (string)
+                + topic (object)
```
Comment (stivni) [inline apiary.apib]: remove for now?

Comment (stivni) [reply apiary.apib]: or rename to work_type

Comment (regularjack) [reply apiary.apib]: :ok: by @regularjack on 29/11/2017 14:44:56

File: apiary.apib (hunk 1 above)
Comment (regularjack) [reply apiary.apib]: Discussed on slack, will be kept as a required filter.

 :ok: by @regularjack on 29/11/2017 14:44:56

**Pull request merged**